lib/driver.js

```diff
@@ -1588,9 +1588,21 @@ class XCUITestDriver extends BaseDriver {
     return true;
   }
 
-  async checkAutInstallationState() {
-    // @ts-expect-error - do not assign arbitrary properties to `this.opts`
-    const {enforceAppInstall, fullReset, noReset, bundleId, device, app} = this.opts;
+  /**
+   * @typedef {Object} CheckAutInstallationState
```

Comment:
CheckAutInstallationState -> AutInstallationState

Type names must not contain verbs